pkg/operator/k8sutil/pod.go

```diff
@@ -51,6 +51,9 @@ const (
 	defaultVersion    = "rook/rook:latest"
 	configMountDir    = "/etc/rook/config"
 	overrideFilename  = "override.conf"
+
+	// DiscoverDaemonUdevBlacklistEnvName discover daemon udev blacklist env name
+	DiscoverDaemonUdevBlacklistEnvName = "DISCOVER_DAEMON_UDEV_BLACKLIST"
```

Comment:
What about removing this https://github.com/rook/rook/blob/master/pkg/daemon/discover/discover.go#L44 and make use of the new constant you just created? 
Or the other way around but you might get an import cycle error.